nsv13/code/modules/overmap/overmap_ghosts.dm

```diff
@@ -73,6 +73,9 @@
 	ghost.hud_type = /datum/hud //Mostly blank hud
 	ghost.key = target.key
 
+	//Allows player to hear hails
+	mobs_in_ship += ghost
```

Comment:
If I recall correctly, this potentially breaks something else. Can't remember what though. vOv.

Reply:
You were right, this caused ghost ships to enter supercritical instead of blowing up (can't believe I forgot to test that). It should be fixed now.